.make/test.mk

```diff
@@ -187,6 +193,32 @@ test-remote-no-coverage: prebuild-check $(SOURCES)
 test-migration: prebuild-check migration/sqlbindata.go migration/sqlbindata_test.go
 	F8_RESOURCE_DATABASE=1 F8_LOG_LEVEL=$(F8_LOG_LEVEL) go test $(GO_TEST_VERBOSITY_FLAG) github.com/fabric8-services/fabric8-wit/migration
 
+define start-wit
+	echo "Starting WIT and ensuring that it is running..."; \
+	F8_LOG_LEVEL=ERROR ./wit+pmcd.sh & 
+	while ! nc -z localhost 8080 < /dev/null; do \
+		printf .; \
+		sleep 5 ; \
+	done; \
+	echo "WIT is RUNNING!";
+endef
+
+.PHONY: test-e2e
+## Runs the end-to-end tests WITHOUT producing coverage files for each package.
+test-e2e: prebuild-check deps docker-compose-up $(SOURCES)
```

Comment:
@rgarg1 since you're trying to start wit in this block, you need to have a target dependency on the `build` target as well. This already includes `deps` and `$(SOURCES)` so you can remove those from here. Also, where do you stop the wit server at the end of all this?